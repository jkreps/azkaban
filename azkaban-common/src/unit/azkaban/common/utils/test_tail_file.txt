Iran Claims to Have Broken Israeli Spy Ring
By WILLIAM YONG and ALAN COWELL	 12:42 PM ET
Iran’s Intelligence Ministry said the network was linked to the assassination of an Iranian nuclear scientist last year.
Iran Sentences Human Rights Lawyer
By WILLIAM YONG	 12:53 PM ET
Nasrin Sotoudeh, a female human rights lawyer, has been sentenced to 11 years in jail for crimes that include “activities against national security” and “propaganda against the regime.”
U.S. Says Sanctions Hurt Iran Nuclear Program
By MARK LANDLER	 11:00 AM ET
Iran’s ability to produce a nuclear weapon has been delayed by sanctions, Secretary of State Hillary Rodham Clinton said.
Iran Claims to Have Broken Israeli Spy Ring 10:28 AM ET
 
Belarus Signals It Could Seize Opponent’s Son
By MICHAEL SCHWIRTZ
The government says it is investigating the status of 3-year-old Danil Sannikov after the arrest of his parents, Andrei Sannikov, a former presidential candidate, and Irina Khalip, a journalist.
 
Afghans Strained by Shortages as Iran Tightens Flow of Fuel
By RAY RIVERA and RUHULLAH KHAPALWAK
The slowdown has forced some gas stations to close and driven up the costs of food and heating oil, just as winter is setting in.
Pakistani Assassin Says He Acted Alone
By SALMAN MASOOD	 11:57 AM ET
The admission by the elite police guard who killed the governor of Punjab Province came at an impromptu court hearing.
Pope Benedict Urges Pakistan to Repeal Blasphemy Law
NATO Strike Kills 3 Afghan Policemen
By RAY RIVERA	 11:14 AM ET
Afghan officials described the airstrike as friendly fire. Separately, two police officers died in a suicide car bomb attack in Kandahar Province.
 
Spain Calls Basque Declaration Insufficient
By RAPHAEL MINDER	 10:37 AM ET
The cease-fire statement by the ETA raised hopes that Spain’s four-decade old fight against the nationalist group could be coming to an end.
In Possible Thaw, North Korea Invites South to Talk
By MARTIN FACKLER	 11:11 AM ET
North Korea proposed working-level talks in late January to prepare the way for a higher level meeting on Feb. 1 to discuss economic ties.
GLOBAL UPDATE
China: Gaps Seen in Government’s Ability to Detect Disease Outbreaks
By DONALD G. McNEIL Jr. 58 minutes ago
China’s ability to identify new outbreaks remains “underdeveloped,” a leading Chinese health official acknowledged in an article in the journal Health Affairs.
 
Floods Kill Four as They Spread South in Australia
By THE ASSOCIATED PRESS	 9:18 AM ET
Rains continued to fall on Queensland, as residents in low-lying communities near Brisbane, Australia’s third-largest city, were told to prepare for flooding.
 
Step by Step, Gulf Between Turkey and Kurds Narrows
By SEBNEM ARSU
Changes, analysts say, are partly the result of Prime Minister Recep Tayyip Erdogan’s longstanding policy of trying to reconcile with the Kurds.
China Suspends Official After Lead Fumes Sicken Children
By MICHAEL WINES
The incident, which left 228 children ill, is the latest in a long series of pollution problems, including many instances of lead poisoning, that have struck rural China in recent years.
 
U.S. and China Defense Chiefs Agree to Keep Talking
By ELISABETH BUMILLER	 5:55 AM ET
The defense secretary Robert M. Gates met with his Chinese counterpart in Beijing, part of a drive to smooth over years of conflict and suspicions.
 
Israeli Demolition Begins in East Jerusalem Project
By ISABEL KERSHNER
Israeli bulldozers demolished part of a landmark building in a predominantly Arab neighborhood to make way for a new Jewish housing project.
