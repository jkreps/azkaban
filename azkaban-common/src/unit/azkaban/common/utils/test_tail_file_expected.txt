Step by Step, Gulf Between Turkey and Kurds Narrows
By SEBNEM ARSU
Changes, analysts say, are partly the result of Prime Minister Recep Tayyip Erdoganâ€™s longstanding policy of trying to reconcile with the Kurds.
China Suspends Official After Lead Fumes Sicken Children
By MICHAEL WINES
The incident, which left 228 children ill, is the latest in a long series of pollution problems, including many instances of lead poisoning, that have struck rural China in recent years.
 
U.S. and China Defense Chiefs Agree to Keep Talking
By ELISABETH BUMILLER	 5:55 AM ET
The defense secretary Robert M. Gates met with his Chinese counterpart in Beijing, part of a drive to smooth over years of conflict and suspicions.
 
Israeli Demolition Begins in East Jerusalem Project
By ISABEL KERSHNER
Israeli bulldozers demolished part of a landmark building in a predominantly Arab neighborhood to make way for a new Jewish housing project.
